src/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+body {
+  background-color: #e5f1f3;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Roboto', sans-serif;
+  font-size: 34px;
+  margin: 0;
+}
+
+h2 {
+  font-size: 1rem;
+}
+
+div#todoHeading {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  border: 1px solid black;
+  padding: 10px;
+}
+
+div#todo {
+  justify-content: space-around;
+}
+
+.fas {
+  font-size: 1rem;
+}
+
+.todo-item {
+  font-size: medium;
+  padding: 10px;
+  flex-direction: row;
+  border: 1px solid black;
+}
+
+div#wrapper {
+  width: 80vw;
+  background-color: #abd8c9;
+  margin: 10vw;
+}
+
+div#addList {
+  justify-content: space-between;
+  flex-direction: row;
+  color: grey;
+  font-style: italic;
+  font-size: medium;
+  border: 1px solid black;
+  padding: 10px;
+}
+
+div#clear {
+  color: white;
+  align-items: center;
+  padding: 20px;
+  font-size: medium;
+  background-color: #58b294;
+}
+
+.checkbox {
+  padding-right: 10px;
+}
+
+.description {
+  flex-grow: 2;
+}
```

Comment:
- kindly, update the background color to stick to the given design and remove style code from the Ui 👍 
<img width="945" alt="2023-07-19_17-57" src="https://github.com/dvcorreia1984/my-to-do-list/assets/52010789/0c367224-d458-4302-98e6-3631562db1ad">